tests/Common/TablesListingTest.php

```diff
@@ -186,6 +186,7 @@ public function testListTablesIncludeColumnMetadata(): void
         $this->assertArrayHasKey('columnMetadata', $firstTable);
         $this->assertArrayHasKey('metadata', $firstTable);
         $this->assertEmpty($firstTable['columnMetadata']);
+        $this->assertIsObject($firstTable['columnMetadata']);
```

Comment:
ono to bude `[]` at uz se v response vrati cokoliv

Reply:
Hmm, ještě je tam nějaký jiný problém...

```
[{"uri":"https:\/\/localhost:8700\/v2\/storage\/tables\/in.c-API-tests.languages","id":"in.c-API-tests.languages","name":"languages","displayName":"languages","transactional":false,"primaryKey":[],"indexType":null,"indexKey":[],"distributionType":null,"distributionKey":[],"syntheticPrimaryKeyEnabled":false,"created":"2023-06-30T11:00:37+0200","lastImportDate":"2023-06-30T11:00:46+0200","lastChangeDate":"2023-06-30T11:00:46+0200","rowsCount":5,"dataSizeBytes":1536,"isAlias":false,"isAliasable":true,"isTyped":false,"tableType":"table","metadata":[],"columnMetadata":[]}]
```

Reply:
Tak som asi skoncil :joy:

Reply:
jj, zkusím to debugnout :(